lib/index.js

```diff
@@ -14,7 +14,7 @@ const configSchema = Joi.object({
           .items(Joi.string().valid('pageview'), Joi.string().valid('event'), Joi.string().valid('ecommerce'))
           .min(1)
           .required()
-      }).required()
+      })
```

Comment:
I think this will allow the key to be omitted completely.

Perhaps re-add the required but do `.min(0).required()`

Reply:
I thought you might be right, but the test is still in place that ensures it can't be undefined:
```
135   it('requires propertySettings to be defined', async () => {
136     await expect(hapiTestServer.start({})).reject('"propertySettings" is required')
137   })
```